Cannot find module (TS2307)

Created nuxt project using `npx`

Contents of tsconfig.json:

```
// tsconfig.json
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "moduleResolution": "node",
    "lib": ["esnext", "esnext.asynciterable", "dom"],
    "esModuleInterop": true,
    "allowJs": true,
    "sourceMap": true,
    "strict": true,
    "noEmit": true,
    "baseUrl": ".",
    "paths": {
      "~/*": ["./*"],
      "@/*": ["./*"]
    },
    "types": ["@types/node", "@nuxt/types"]
  },
  "exclude": ["node_modules"]
}
```

Index.vue (where i face import issue):

```
<script lang="ts">
import Logo from '~/components/Logo.vue'

export default {
  components: {
    Logo
  }
}
</script>
```
Component successfully imports, but TS server reports an error `Error:(28, 18) TS2307: Cannot find module '~/components/Logo.vue'.`

--- a/docs/guide/setup.md
+++ b/docs/guide/setup.md
@@ -62,6 +62,16 @@ and create a **`tsconfig.json`** file :
 }
 ```
 
+You will also need to provide types for Vue files by adding the following type declaration:
+
+`vue-shim.d.ts`:
+```ts 
+declare module "*.vue" {
+  import Vue from 'vue'
+  export default Vue
+}
+```
+
 ::: tip
 `@nuxt/typescript-build` ships `@nuxt/types`, so there's no need to install it independently.
 :::

